modules/ocf_puppet/manifests/puppetserver.pp

```diff
@@ -39,12 +39,12 @@
     notify               => Service['puppetserver'],
   }
 
-  # Allow kubernetes masters to access kubernetes secrets
+  # Allow Jenkins to access kubernetes secrets
   puppet_authorization::rule { 'kubernetes-secrets':
     match_request_path   => '^/puppet/v3/file_(content|metadata)s?/kubernetes-secrets$',
     match_request_type   => 'regex',
     match_request_method => ['get'],
-    allow                => suffix(lookup('kubernetes::master_nodes'), '.ocf.berkeley.edu'),
+    allow                => 'reaper.ocf.berkeley.edu',
```

Comment:
Does this work with `jenkins.ocf.berkeley.edu` instead, or does it have to be the fqdn of the host? It would be better if this was the service name to be more generic rather than the specific hostname of the jenkins host.

Reply:
lol abizer just mentioned this too, guess I'm slow

Reply:
It needs to be the fqdn, I tested jenkins.ocf.berkeley.edu first